app/models/repo.rb

```diff
@@ -70,7 +70,9 @@ def with_travis_service
   end
 
   def enabled_for?(checker)
-    Array(Settings.public_send(checker).enabled_repos).include?(name)
+    Array(Settings.public_send(checker).enabled_repos).each_with_object([]) { |value, values|
+      values << (value.kind_of?(Array) ? value.first.to_s : value)
+    }.include?(name)
```

Comment:
Oh hah, I could see that. In my [lesser] experience I just don't see people thinking to use reduction and never see anyone using it. Fair.

Anyway, leaving this here given my point above since this seems a valid use case, otherwise if you still insist we can just use moar intermediate lines.